### PR TITLE
Calling global Browserify in build

### PR DESCRIPTION
**Pull request by RongxinZhang**
**Issue:**
- Browserify is being called globally on post install. Never do that.

**Solution:**
- [x] move browserify out of dev dependencies 

ref: https://github.com/Retsly/www/issues/556
ref: https://github.com/Retsly/api/issues/1515

### CODE REVIEW

**Comment by RongxinZhang** [comment]
@thalesfsp @lvivier plz review

File: wercker.yml [hunk 1]
```diff
@@ -4,14 +4,20 @@ build:
     - script:
         name: chown for npm
         code: sudo chown -R $USER /usr/local
+
     - script:
-        name: github auth
+        name: install newer npm
```
Comment (thalesfsp) [inline wercker.yml]: Update to: "install npm 2.14.8" newer mean, "the most new"? ;)

**Comment by thalesfsp** [comment]
:+1:

**Comment by lvivier** [comment]
:+1: :+1: :+1: :+1:

**Pull request merged**